src/services/mongoose/wallet-invoices.ts

```diff
@@ -144,7 +142,7 @@ export const WalletInvoicesRepository = (): IWalletInvoicesRepository => {
 }
 
 const walletInvoiceFromRaw = (result: WalletInvoiceRecord): WalletInvoice => ({
-  paymentHash: result.id as PaymentHash,
+  paymentHash: result._id as PaymentHash,
```

Comment:
If `_id` gets typed as `ObjectId` from earlier comment, this may changes as well to `fromObjectId...`

Reply:
N/A https://github.com/GaloyMoney/galoy/pull/1406#discussion_r924628569